bin/Powheg/production/Run3/13p6TeV/bb_H_2HDM_13p6TeV/CardSpace/createCards.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+for mass in 50 60 65 70 75 80 85 90 95 100 105 110 115 120 125 130 135 140 160 180 200 250 300 350 400 450 500 600 700 800 900 1000 1100 1200 1400 1600 1800 2000 2300 2600 2900 3200 3500;
+do
+
+    for hdampmode in "" 1 2; 
+    do 
+    
+        if (( $(echo $mass'<'145 | bc -l) )); 
+        then 
+            width=0.0041
+        elif (( $(echo $mass'<'605 | bc -l) )); then 
+            width=0.1
+        elif (( $(echo $mass'<'2005 | bc -l) )); then
+            width=1.0
+        else
+            width=2.0
```

Comment:
for the width, are those number given by some phone paper or randomly, will you use the cross section connects to this MC process in your analysis

Reply:
We are using these numbers inheranted from the published Run2 Analysis ([HIG-21-001](https://cms.cern.ch/iCMS/analysisadmin/cadilines?line=HIG-21-001&tp=an&id=2409&ancode=HIG-21-001)).

Maybe @danielwinterbottom could provide more references or correct me.